src/migrate/sources/fs-migrations.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+import Promise from 'bluebird';
+import { sortBy } from 'lodash';
+
+const readDirAsync = Promise.promisify(fs.readdir, { context: fs });
+
+export class FsMigrations {
```

Comment:
Why not export it as a default?

Reply:
Personal preference coming from TypeScript. https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

Happy to change if that is the preference.

Reply:
It result in slightly less redundant imports, 
Quick search on project shows that there are multiple cases of "export default class" and none of "export class", so it does seem to be the project convention.

Reply:
Will change